bin/blin.p6

```diff
@@ -379,6 +382,27 @@ sub save-overview {
 save-overview;
 
 
+note '🥞🥞 Saving the markdown output';
+sub save-markdown {
+  $save-lock.protect: {
+    my $markdown-output = '';
+
+    for @sorted-modules {
+      my $bisected = '';
+
+      next if (my $result = .done ?? .done.result !! Unknown) == OK;
```

Comment:
I think this should be `== Bisected` because other types of failures are very common. For example, modules that simply can't pass their test end up with `AlwaysFail` result, and there are hundreds of them. For Rakudo release process we only care about those that worked fine previously but no longer do, so that's ``Bisected``.

Reply:
Also I believe we can get away with not checking if promises are ready (as we'll be running this at the end of the program).

Reply:
So... `next unless (my $result = .done ?? .done.result !! Unknown) == Fail;` ?

You were talking about an if statement, so I was confused for a second.

Reply:
I think just `.done.result == Fail` should work but I'm not 100% sure.